translations/generate.plugin.rulesaction.yml

```diff
@@ -1,29 +0,0 @@
-description: 'Genera un plugin de acción de rules'
```

Comment:
@novia713 why this file was deleted?

Reply:
seems in english the file is with dot also, so i guess this deleting and creating the new file with dot is correct ....

Reply:
@jmolivas do you have any comments here?

Reply:
We should use same as en language

Reply:
We can take a look at the command code to check the correct one